incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -46,25 +46,50 @@ func RunShouldContain(substr string, seconds float64, cmdln ...string) {
 
 func RunShouldContainMultiple(substrs []string, seconds float64, cmdln ...string) {
 	Eventually(func() error {
-		stdout, err := RunCommand(cmdln...)
-		if err != nil {
+		return runShouldContainMultiple(substrs, cmdln...)
+	}, seconds).Should(Succeed())
+}
+
+func RunErrorShouldContain(substr string, seconds float64, cmdln ...string) {
+	RunErrorShouldContainMultiple([]string{substr}, seconds, cmdln...)
+}
+
+func RunErrorShouldContainMultiple(substrs []string, seconds float64, cmdln ...string) {
+	Eventually(func() error {
+		err := runShouldContainMultiple(substrs, cmdln...)
+		GinkgoT().Log("Error from: RunErrorShouldContainMultiple ", err)
+		if containAll([]string {"Expecting: ", " but get: "}, err.Error()) {
```

Comment:
Just confirming, the `substrs []string` are the ones should appear in the error message, right? Seems the `err` on line 59 should come from just running the command instead of caring about whether `substrs` are there or not. Then line 61 should see if the error msg contains all `substrs`. CMIIW.

Reply:
`substrs []string` are the ones should appear in the `stdout` from running the cmd, which is checked in `runShouldContainMultiple`. Here we just want to distinguish if the `err` comes from 
    1. `substrs` not found in `stdout`
    2. running `cmd` returns an error

If the `err` contains `"Expecting: ", " but get: "`, then it means `stdout` does not contain the `substrs` we want, so it returns an err. Otherwise, the `err` comes from running the cmd, which is what we are expecting

Reply:
Thanks for the explanation! The new change LGTM!